### DISCUSSION TITLE
"Nestled" snippet tab stops

### DISCUSSION THREAD

**Discussion by Oozna** [draivin/hsnips#123] [Q&A] [answered]
Thank you for developing this extension, I love it!
I have a question about if it's possible to not remove the "outer" snippet's tab-stop when calling a new one?
Inserting video for clarification.

![snippetgif](https://user-images.githubusercontent.com/66669398/170984770-fb67fc6e-bb11-49b1-afd5-027de1d6cd12.gif)

(PS I know this code is nonsense but is just for demonstration)
Desired behaviour is to be able to tab to the $2 (next line) of the "cout" after the "veci" is inserted.

Here are the snippets:
"veci"
<img width="321" alt="Screenshot 2022-05-30 at 13 34 11" src="https://user-images.githubusercontent.com/66669398/170984199-829660fe-d4aa-4671-b3f7-3f98aa6a30f2.png">

"cout"
<img width="333" alt="Screenshot 2022-05-30 at 13 34 38" src="https://user-images.githubusercontent.com/66669398/170984219-c971bd2b-a2d4-4316-835e-ee67a78197f5.png">

I've looked around for a while but haven't found/understood a solution yet.
If there is no solutions that's fine too

**Comment by yfzhao20** [comment, marked as answer]
![test](https://user-images.githubusercontent.com/80528249/170993718-4d33c48e-810c-4cc3-8e19-5d6eea462eb0.gif)

I cannot confirm this issue...try  disabling other extensions ?

**Reply by Oozna** [reply]
Thank you! After doing an extension-bisect it seems to be an issue/conflict with the "Vim" extension, I'll look into it more. 
Dumb of me to not having checked it before asking the question

**Reply by yfzhao20** [reply]
That's okay! And you may also check #78. It reported a similar issue about VIM √

 Unfortunately I don't know how to use Vim and could not give some help..

**Reply by Oozna** [reply]
Thank you I didn't find that post when looking myself. 
I changed to another vim-extension called "amVim" which does not seem to have this problem.
